loops-and-arrays.js

```diff
@@ -31,8 +43,22 @@
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 getallen zou bevatten!
 // ==========================================
 
+const squares = [30, 2, 8, 24, 11];
+console.log(squares);
+for (k=0; k < squares.length; k++){
+  console.log(squares[k] * squares[k] * squares[k]);
+}
+
 // Verwachtte uitkomsten:
 // Vóór het script zie je de originele waardes:
 // console.log(numbers) geeft: [30, 2, 8, 24, 11];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(numbers) geeft: [27000, 8, 512, 13824, 1331];
+
```

Comment:
Hier heb je het wel gedaan zoals de opdracht omschreef maar dit als comment er in gezet.